File: rb/company.rb
class Company
  attr_accessor :employees, :customers

  def initialize
    @employees = []
    @customers = []
  end

  def add_employee(employee)
    @employees << employee
  end

  def list_employees
    @employees.each do |employee|
      puts "#{employee.name} <#{employee.email}>  Phone: #{employee.phone} Salary: $#{'%.2f' % employee.salary}"
    end
  end
end

File: Lua/Lua/company.lua
-- -- Module with Company class that returns Company table
Company = {}
Company.__index = Company

function Company.new()
    local self = setmetatable({}, Company)
    self.employees = {}
    self.customers = {}
    return self
end

function Company:add_employee(employee)
    table.insert(self.employees, employee)
end

function Company:list_employees()
    for _, employee in ipairs(self.employees) do
        print(employee.name .. " <" .. employee.email .. ">  Phone: " .. employee.phone .. " Salary: $" .. string.format("%.2f", employee.salary))
    end
end

return Company